### PR TITLE
remove code smells

### PR DESCRIPTION
**Pull request by Bodasyuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs [hunk 1]
```diff
@@ -215,13 +216,16 @@ public async Task<IEnumerable<SectorAdministrationDTO>> GetAdministrationsOfUser
...
+            admins.Where(a => a.Sector != null).ForAll(a => a.Sector.Administration = null);
+
+            //var administrations = admins.ToList();
+            //foreach (var admin in administrations)
+            //{
+            //    if (admin.Sector != null)
+            //    {
+            //        admin.Sector.Administration = null;
+            //    }
+            //}
```
Comment (DmytroZelinsky) [inline EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs:228]: delete the commented code

Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs]: that was test

File: EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs [hunk 2]
```diff
@@ -11,6 +11,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using AutoMapper.Internal;
```
Comment (VolodymyrKostiv) [inline EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs:14]: Sort usings

**Review by petromonka — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs [hunk 3]
```diff
@@ -233,15 +227,16 @@ public async Task<IEnumerable<SectorAdministrationDTO>> GetPreviousAdministratio
                  source => source.Include(c => c.User).Include(c => c.AdminType).Include(a => a.Sector)
                  );
 
-            foreach (var admin in admins)
+            var administrations = admins.ToList();
+            foreach (var admin in administrations)
```
Comment (VolodymyrKostiv) [inline EPlast/EPlast.BLL/Services/GoverningBodies/Sector/SectorService.cs:231]: Check sonarcloud, seems like there is an issue with this loop

File: EPlast/EPlast.DataAccess/Repositories/Realizations/City/CityRepository.cs [hunk 4]
```diff
@@ -1,5 +1,8 @@
-﻿using EPlast.DataAccess.Entities;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using EPlast.DataAccess.Entities;
 using EPlast.DataAccess.Repositories.Contracts;
+using Microsoft.EntityFrameworkCore;
```
Comment (VolodymyrKostiv) [inline EPlast/EPlast.DataAccess/Repositories/Realizations/City/CityRepository.cs:5]: Sort usings (sorry, I didn't see it the first time)

File: EPlast/EPlast.DataAccess/Repositories/Interfaces/City/ICityRepository.cs [hunk 5]
```diff
@@ -1,4 +1,6 @@
-﻿using EPlast.DataAccess.Entities;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using EPlast.DataAccess.Entities;
```
Comment (VolodymyrKostiv) [inline EPlast/EPlast.DataAccess/Repositories/Interfaces/City/ICityRepository.cs:3]: Sort usings

**Review by VolodymyrKostiv — APPROVED** [review]

**Review by IlliaDR — APPROVED** [review]

**Pull request merged**